data/mships.txt

```diff
@@ -1430,7 +1430,7 @@ ship "FG-5 Bluewhale"
 	explode "large explosion" 35
 	explode "huge explosion" 20
 	"final explode" "final explosion large"
-	description "Bluewhale is a massive freighter for hauling large amount of cargo across the galaxy. Due to its' cost these ship are often owned by freight companies rather than individuals. While it's capable of carrying much more cargo than most other freighters, one of the major complaint received from captains is that it can take very long to reach the innermost container. That said it is still far faster and easier than handling of bulk cargo."
+	description "The Bluewhale is a massive freighter for hauling large amount of cargo across the galaxy. Due to its cost these ships are often owned by freight companies rather than individuals. While its capable of carrying much more cargo than most other freighters, one of the major complaints received from captains is that it can take very long to reach the innermost container. That said it is still far faster and easier than handling of bulk cargo."
```

Comment:
```suggestion
	description "The Bluewhale is a massive freighter for hauling large amount of cargo across the galaxy. Due to its' cost these ships are often owned by freight companies rather than individuals. While it's capable of carrying much more cargo than most other freighters, one of the major complaints received from captains is that it can take very long to reach the innermost container. That said it is still far faster and easier than handling of bulk cargo."
```